bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/Layer/Layer.jsx

```diff
@@ -55,7 +56,9 @@ const Layer = ({ model, even, selected, controller }) => {
                         .filter(tool => tool.getTypes().includes('layerList'))
                         .map((tool, i) =>
                             <Tooltip key={`${tool.getName()}_${i}`} title={tool.getTooltip()}>
-                                <div className={tool.getIconCls()} onClick={() => onToolClick(tool)}/>
+                                <div onClick={() => onToolClick(tool)}>
+                                    <EditFilled />
```

Comment:
Use tool.getIconComponent() here as well so we don't need to bundle the SVG if it's not used for non-admin users and it allows multiple tools with different icons to be added.

Reply:
Done